src/server/rest/v1/service/ChargingStationService.ts

```diff
@@ -1221,9 +1226,70 @@ export default class ChargingStationService {
           action: action
         });
       }
+      const tag = await TagStorage.getTagByVisualID(req.tenant.id, filteredRequest.args.visualTagID, { userIDs: [user.id] });
+      if (!tag) {
+        throw new AppError({
+          source: Constants.CENTRAL_SERVER,
+          errorCode: HTTPError.USER_NO_BADGE_ERROR,
+          message: 'The user does not have any badge',
```

Comment:
Message should contain more info:
The user does not own a badge with ID '<BadgeID>'

Reply:
Done.